### DISCUSSION TITLE
Cisco 'Secure Firewall' 3100 series templates?

### DISCUSSION THREAD

**Discussion by TheDirkMelvin** [netbox-community/devicetype-library#1222] [Ideas]
I have not recognized a roll out pattern for newer devices being added as templates here, but we now have some Cisco Secure Firewall 3100 series but I don't see a template for any of them here so far. They have been out for over a year now.

If no one has a template already, meaning I will have to manually create one, I would be happy to contribute for 3110 and 3120 models. Once I actually create them, that is. :)

Thanks,
Dirk

**Comment by danner26** [comment]
Hey @TheDirkMelvin, if the template does not currently exist please feel free to create a PR with the relevant information. Once the PR is submitted for review we will go ahead and merge it after the review is approved. If you need any assistance please let me know!
